app/components/PortfolioList.tsx

```diff
@@ -37,7 +37,7 @@ function PortfolioList() {
     }
   }
 
-  function findPercentPriceChange(p, c) {
+  function findPercentPriceChange(p: number, c: number) {
```

Comment:
i would make the p have some meaning, I assume price and c should stand for change or current